demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -1064,6 +1185,17 @@ static int subscribePublishLoop( MQTTContext_t * pMqttContext,
         }
     }
 
+    if( ( returnStatus == EXIT_SUCCESS ) && ( globalSubscribeStatus == MQTTSubAckFailure ) )
+    {
+        /* If SUBACK is decoded as server rejection (in function updateSubscribeStatus, called by eventCallback),
+         * attempt to resubscribe to topic. Attempts are made according to the exponential backoff
+         * retry strategy implemented in transport_reconnect. */
```

Comment:
The comment can be simplified to mention
```suggestion
        /* If server rejected the subscription request, attempt to resubscribe to topic. 
          * Attempts are made according to the exponential backoff retry strategy 
           * implemented in transport_reconnect. */
```